src/api/index.js

```diff
@@ -114,6 +114,11 @@ export function update(links, id, value, name, config) {
 
 export function submit(href) {
   const data = state.get().drafts[href]
-  requestResource({ data, href, method: 'post', resourceKey: 'current' })
-  state.get().drafts.remove(href)
+  requestResource({
+    data,
+    href,
+    method: 'post',
+    resourceKey: 'current',
+    onDone: () => state.get().drafts.remove(href),
```

Comment:
The draft needs to be cleared only if there was no error on the request.
What I understand is that `requestResource` just queues the request to be processed later, it is a sync call and doesn't return the result of the request.

How should I use `await` here?